src/internal-links/handler.js

```diff
@@ -67,64 +39,141 @@ function calculatePriority(links) {
  * @returns {Response} - Returns a response object indicating the result of the audit process.
  */
 export async function internalLinksAuditRunner(auditUrl, context) {
-  const { log } = context;
+  const { log, site } = context;
   const finalUrl = await getRUMUrl(auditUrl);
 
-  const rumAPIClient = RUMAPIClient.createFrom(context);
+  try {
+    const rumAPIClient = RUMAPIClient.createFrom(context);
 
-  const options = {
-    domain: finalUrl,
-    interval: INTERVAL,
-    granularity: 'hourly',
-  };
-
-  log.info(`broken-internal-links audit: ${auditType}: Options for RUM call: `, JSON.stringify(options));
-
-  const internal404Links = await rumAPIClient.query('404-internal-links', options);
-  const transformedLinks = internal404Links.map((link) => ({
-    urlFrom: link.url_from,
-    urlTo: link.url_to,
-    trafficDomain: link.traffic_domain,
-  }));
+    const options = {
+      domain: finalUrl,
+      interval: INTERVAL,
+      granularity: 'hourly',
+    };
 
-  let finalLinks = calculatePriority(transformedLinks);
+    log.info(
```

Comment:
is this for debug? please remove any log statements that are not needed for production analysis, in order to save log volume on coralogix